src/data/mv_polynomial/pi.lean

```diff
@@ -0,0 +1,62 @@
+/-
+Copyright (c) 2022 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Junyan Xu, Andrew Yang
```

Comment:
@alreadydone, I've added you as an author for this file if you don't mind it.

Reply:
Thanks but it seems we no longer need a new file?